ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -213,22 +213,22 @@ def conv_transpose1d(
 def conv_transpose2d(
     input,
     weight,
-    bias=None,
-    stride=1,
-    padding=0,
-    output_padding=0,
-    groups=1,
-    dilation=1,
+    strides=1,
+    padding="SAME",
+    output_shape=None,
+    data_format="NHWC",
+    dilations= 1,
+    bias= None,
 ):
-    return _conv_transpose(
```

Comment:
And since the ivy function expects somewhat different arguments, extra operations are needed before calling it.